packages/sources/alphavantage/README.md

```diff
@@ -40,10 +40,10 @@ Supported names for this endpoint are: `forex`, `price`.
 
 ### Input Params
 
-| Required? | Name  |    Aliases     |                  Description                   |  Type  | Options | Default | Depends On | Not Valid With |
-| :-------: | :---: | :------------: | :--------------------------------------------: | :----: | :-----: | :-----: | :--------: | :------------: |
-|    ✅     | base  | `coin`, `from` | The symbol of symbols of the currency to query | string |         |         |            |                |
-|    ✅     | quote | `market`, `to` |    The symbol of the currency to convert to    | string |         |         |            |                |
+| Required? |    Name    |               Aliases                | Description | Type | Options | Default | Depends On | Not Valid With |
+| :-------: | :--------: | :----------------------------------: | :---------: | :--: | :-----: | :-----: | :--------: | :------------: |
+|           | definition |                                      |             |      |         |         |            |                |
+|    ✅     |   params   | `[object Object]`, `[object Object]` |             |      |         |         |            |                |
```

Comment:
Input params generation is broken - likely due to using a different version of `ea-framework-js` in `scripts` and the EAs. This looks to trip up the [endpointDetails.inputParameters instanceof V3InputParameters ](https://github.com/smartcontractkit/external-adapters-js/blob/main/packages/scripts/src/generate-readme/generator.ts#L434)check.